cmd/skupper/skupper_kube_network.go

```diff
@@ -36,6 +37,14 @@ func (s *SkupperKubeNetwork) Status(cmd *cobra.Command, args []string) error {
 		fmt.Println()
 		return nil
 	}
+
+	configSyncVersion := utils.GetVersionTag(s.kube.Cli.GetVersion(types.TransportContainerName, types.ConfigSyncContainerName))
+	if configSyncVersion != "" && !utils.IsValidFor(configSyncVersion, network.MINIMUM_VERSION) {
+		fmt.Printf("Site version is %s, but CLI requires %s", configSyncVersion, network.MINIMUM_VERSION)
```

Comment:
Propose something similar to the following to suggest to user action to take:

"Detected that the skupper version installed in the namespace is version %s. The CLI requires version %s. To update the installation, please follow the instructions found here"

where here is link to skupper docs

Reply:
Updated in f8197e767e2ae05ed76fd5debeae3bda0bf0bca1